aip-console-tools-cli/src/test/java/com/castsoftware/aip/console/tools/AipConsoleToolsCliIntegrationTest.java

```diff
@@ -0,0 +1,57 @@
+package com.castsoftware.aip.console.tools;
+
+import com.castsoftware.aip.console.tools.commands.ParentCommand;
+import com.castsoftware.aip.console.tools.core.utils.Constants;
+import com.castsoftware.aip.console.tools.factories.SpringAwareCommandFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.ExitCodeGenerator;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import picocli.CommandLine;
+
+//@SpringBootApplication
```

Comment:
Please remove this, no point in keeping an annotation in a comment

Reply:
Will be done